gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -70,11 +70,19 @@
 ]
 # If the # of files matching the input file_pattern exceeds this value, then
 # headers will be merged in beam.
-_USE_HEADER_MERGER_THRESHOLD = 100
+_USE_BEAM_READ_THRESHOLD = 100
```

Comment:
let's rename these to '_SMALL_DATA_THRESHOLD', '_LARGE_DATA_THRESHOLD'

Reply:
Done.